examples/keyvalue/Makefile

```diff
@@ -7,4 +9,7 @@ server/server: $(wildcard server/*.go) kv/types.go
 .PHONY: all
 all: server/server
 
-.DEFAULT_GOAL: all
+.PHONY: deps
+deps:
+	$(ECHO_V)go get -v go.uber.org/thriftrw
+	$(ECHO_V)go get -v go.uber.org/yarpc/encoding/thrift/thriftrw-plugin-yarpc
```

Comment:
This is wrong. We need to use the version that matches what we have in glide.yaml, otherwise the type signatures are going to mismatch. The YARPC team noticed and fixed this for us while you were away in december.